src/lib.rs

```diff
@@ -26,10 +26,6 @@
 #![deny(missing_copy_implementations, safe_packed_borrows)]
 #![cfg_attr(not(feature = "rustc-dep-of-std"), no_std)]
 #![cfg_attr(feature = "rustc-dep-of-std", no_core)]
-#![cfg_attr(
-    feature = "rustc-dep-of-std",
-    feature(native_link_modifiers, native_link_modifiers_bundle)
-)]
```

Comment:
Could you explain why this change is necessary?

Reply:
`native_link_modifiers` and `native_link_modifiers_bundle` are already stable, so these features produce `#[warn(stable_features)]` in `rustc-dep-of-std` mode (which turns into an error when building standard library in rust-lang/rust).